arborescence.md

```diff
@@ -38,16 +38,18 @@
 # Arboresence typique des tests d'intégration
 
 - [X] Makefile
-- [ ] Dockerfile-*
+- [ ] docker
+  - [ ] Dockerfile-*
```

Comment:
What is the point to add the directory structure `docker`? I though we had decided to do not add it since the file in the directory already contains the the word `Dockerfile`

Reply:
I'd agree if there were only Dockerfiles. But in some cases, there are files needed for the container image, like [entrypoint.sh](https://github.com/wazo-platform/wazo-auth-keys/tree/master/integration_tests). So I prefer to store all container-related files in a separate directory.

Reply:
Ok thanks for the explanation